routers/posts.py

```
@@ -1,29 +1,278 @@
+                }
+            }
+        )
+    # 게시글 데이터 로드
+    posts = load_data("posts")
+    users = load_data("users")
+    #postId 생성
+    new_post_Id = (
+        max([p["postId"] for p in posts], default=0) + 1
+    )
+    #현재 시간
+    created_at = datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)